src/amo/components/GetFirefoxButton/styles.scss

```diff
@@ -10,3 +13,47 @@
     width: auto;
   }
 }
+
+// New styles for the CTA experiment
+.GetFirefoxButton--new {
+  .GetFirefoxButton-button {
+    margin: auto;
+    white-space: nowrap;
+    width: 100%;
+
+    @include respond-to(medium) {
+      align-self: flex-end;
```

Comment:
Is it working okay in RTL mode?

Reply:
It is! I guess it makes sense that `flex-end` would be language-direction-aware, as it's flex-*end*, not flex-*right*.